app/idValidation.go

```
@@ -0,0 +1,14 @@
+package app
+
+import "strconv"
+
+func ValidateID(index string) int {
+
+	wantedIndex, _ := strconv.Atoi(index)
```

Please don't ignore the error that functions return, we should handle all of them.

By handling I mean, logging it and returning either the error or a custom one